classes/Hook/HookDisplayHeader.php

```diff
@@ -100,12 +104,18 @@ public function setParams($params)
     }
 
     /**
-     * setBackOffice
-     *
-     * @param array $backOffice
+     * @param bool $backOffice
      */
     public function setBackOffice($backOffice)
     {
-        $this->module->backOffice = $backOffice;
+        $this->acknowledgeBackOfficeContext($backOffice);
+    }
+
+    /**
+     * @param bool $isBackOffice
+     */
+    public function acknowledgeBackOfficeContext($isBackOffice)
```

Comment:
I think function name `setBackOffice` is ambiguous, which is why I provide a replacement. I keep the old function for backward compatibility.